include/fastscapelib/structured_grid.hpp

```diff
@@ -330,6 +330,8 @@ namespace fastscapelib
 
         const node_status_type& status_at_nodes() const;
 
+        distance_type node_area(const size_type& idx) const noexcept;
```

Comment:
```suggestion
        distance_type cell_area(const size_type& idx) const noexcept;
```

Could we also create a `area_type` alias? It's another dumb alias, but `distance_type` sounds very odd for a cell area.

Reply:
Sure :)

Reply:
Let's improve this in a new PR if it's OK for you

Reply:
Yes sure.